### ISSUE TITLE
cql_additional_tests.py:TestCQL.null_support_test regression

### ISSUE THREAD

**Issue by bhalevy** [scylladb/scylla#7812]
The tests started failing in https://jenkins.scylladb.com/view/master/job/scylla-master/job/dtest-release/695/testReport/cql_additional_tests/TestCQL/null_support_test/
```
Error Message
Expecting query to be invalid: got an ok response <cassandra.cluster.ResultSet object at 0x7fb027639810>
Stacktrace
Traceback (most recent call last):
  File "/usr/lib64/python3.7/unittest/case.py", line 60, in testPartExecutor
    yield
  File "/usr/lib64/python3.7/unittest/case.py", line 645, in run
    testMethod()
  File "/jenkins/workspace/scylla-master/dtest-release/scylla-dtest/cql_additional_tests.py", line 1265, in null_support_test
    assert_invalid(session, "SELECT * FROM test WHERE k = null")
  File "/jenkins/workspace/scylla-master/dtest-release/scylla-dtest/assertions.py", line 41, in assert_invalid
    raise e
  File "/jenkins/workspace/scylla-master/dtest-release/scylla-dtest/assertions.py", line 39, in assert_invalid
    assert False, "Expecting query to be invalid: got an ok response %s" % str(res)
AssertionError: Expecting query to be invalid: got an ok response <cassandra.cluster.ResultSet object at 0x7fb027639810>
```

After a60c81b6154a5972c6ead65c657f4c28ebdea7c6
(in particular it looks like 4bb11076525df443d155e45331c82a3f34e2e649 caused the regression.)

@dekimir please see who's at fault, scylla or the test?

**Comment by dekimir** [comment]
@bhalevy I'm on vacation until Jan 4. This is almost certainly an incorrect test duplicating what unit tests already cover. All such tests are trouble and should be deleted.

**Comment by bhalevy** [comment]
> @bhalevy I'm on vacation until Jan 4. This is almost certainly an incorrect test duplicating what unit tests already cover. All such tests are trouble and should be deleted.

@dekimir enjoy your vacation. I tend to agree, but this can wait until you're back.

**Comment by bhalevy** [comment]
Fixed by scylladb/scylla-dtest@37af46696e02ab9fbcd96dcabe923472f26233ac
